app/models/solution/mentor_discussion.rb

```diff
@@ -38,4 +38,12 @@ def viewable_by?(user)
     # TODO: Admins should also be allowed to view
     [mentor, student].include?(user)
   end
+
+  def mark_as_nothing_to_do(user)
+    update(requires_mentor_action_since: nil) if user == mentor
```

Comment:
Use `update_column` as we don't care about validations.
Also, let's remove this guard. The guard should be external to the model, either in a command or the controller.

```suggestion
    update_column(:requires_mentor_action_since, nil) 
```